docs/jazzy.yml

```diff
@@ -22,68 +22,108 @@ custom_categories:
       - NavigationViewController
       - NavigationViewControllerDelegate
       - RouteVoiceController
+      - VoiceControllerDelegate
+      - MapboxVoiceController
+      - NavigationOptions
+      - NavigationView
   - name: Core Navigation Logic
     children:
       - NavigationService
+      - NavigationServiceDelegate
       - MapboxNavigationService
+      - Router
+      - RouterDataSource
       - RouteController
-      - RouteControllerDelegate
+      - RouterDelegate
       - RouteProgress
       - RouteLegProgress
       - RouteStepProgress
-      - RouteControllerNotificationUserInfoKey
+      - NavigationEventsManager
+      - EventsManagerDataSource
+      - NavigationLocationManager
+      - ReplayLocationManager
+      - SimulatedLocationManager
+      - NavigationRouteOptions
+      - NavigationMatchOptions
+      - NavigationDirections
+      - SimulationMode
   - name: Maps
     children:
       - NavigationMapView
       - NavigationMapViewDelegate
       - MGLMapView
   - name: Styling
     children:
+      - LanesStyleKit
+      - ManeuversStyleKit
       - Style
-      - StyleKitArrows
-      - StylableButton
       - StylableLabel
+      - StylableView
+      - StylableTextView
+      - StylableButton
```

Comment:
These classes are `:nodoc:`’d because they’re intentionally left undocumented: the NSAppearance system requires them to be public, but developers unknowingly use them as proxy objects when configuring the appearance of the associated views.